pre-docs/bolt_command_reference.md

```diff
@@ -1,109 +1,498 @@
-| `bolt puppetfile install` | Installs the modules listed in the Puppetfile to the current `Boltdir`. | |
-| `bolt puppetfile show-modules` | Lists the modules, and their versions, that are installed in the current `Boltdir`. | |
-| `bolt inventory show` | Show the list of targets an action would run on. | - The inventory targets to show. |
+## Global Options
 
-## Command options
+The following options are available for all subcommands and actions.
 
-Options are optional unless marked as required. 
+| Option | Description |
+|--------|-------------|
+| `-h`, `--help` | Display the help text. |
+| `--version` | Display the version of Bolt. |
+| `--debug` | Display debug logging. |
 
-|Option|Description|
-|------|-----------|
-|`--nodes`, `-n` | **Required****when running**. Nodes to connect to.
 
- To connect with WinRM, include the protocol as `winrm://<HOSTNAME>`.
+## `apply`
 
- For an IPv6 address without a port number, encase it brackets `[fe80::34eb:ff1:b584:d7c0]`.
+Apply a Puppet manifest file.
 
- For IPv6 addresses including a port use one of the following formats:  `fe80::34eb:ff1:b584:d7c0:22 ` or  `[fe80::34eb:ff1:b584:d7c0]:22`.
+### Usage
 
- |
-| `--query` `, -q` |Query PuppetDB to determine the targets.|
-|`--rerun`| use the `.rerun.json` file to choose the targets. Requires a single filter options: all, failure, or success.
-| `--noop` |Execute a task that supports it in no-operation mode.|
-| `--description` |Add a description to the run. Used in logging and submitted to Orchestrator with the PCP transport.|
-| `--params` | Parameters, passed as a JSON object on the command line, or as a JSON parameter file, prefaced with `@` like `@params.json`. For Windows PowerShell,  add single quotation marks to define the file: `'@params.json'`
+`bolt apply <MANIFEST> <TARGETS>`
 
- |
+- You must specify one of `--nodes`, `--targets`, `--query`, or `--rerun`.
 
-## Authentication options
+### Options
 
-|Option|Description|
-|------|-----------|
-|`--user`, `-u`|User to authenticate as.|
-|`--password`, `-p`|Password to authenticate with. Pass this flag without any value to securely prompt for the password.|
-| `--private-key` |Private ssh key to authenticate with|
-| `--host-key-check, --no-host-key-check` | Do not require verification of new hosts in the `known_hosts` file.
+| Option | Description | Default |
+|--------|-------------|---------|
+| `-n`, `--nodes NODES` | Alias for `--targets`. |
+| `-t`, `--targets TARGETS` | Identifies the targets of command. |
+| `-q`, `--query QUERY` | Query PuppetDB to determine the targets. <br> Enter a comma-separated list of target URIs or group names. Or read a target list from an input file `@<file>` or stdin `-`. |
+| `--rerun FILTER` | Retry on nodes from the last run. <br> `all` runs on all targets from the last run. <br> `failure` runs on all targets that failed in the last run. <br> `success` runs on all targets that succeded in the last run. |
```

Comment:
s/succeded/succeeded/g I would make the suggestion in the UI, but it looks like the typo got copied around. Will likely be easiest to just do it in your text editor.